pkg/actions/karpenter/create.go

```diff
@@ -27,18 +27,21 @@ func (i *Installer) Create() error {
 			return i.ClientSet, nil
 		},
 	}
-	karpenterServiceAccountTaskTree := i.StackManager.NewTasksToCreateIAMServiceAccounts([]*api.ClusterIAMServiceAccount{
-		{
-			ClusterIAMMeta: api.ClusterIAMMeta{
-				Name:      karpenter.DefaultServiceAccountName,
-				Namespace: karpenter.DefaultNamespace,
+	i.Config.SetDefaultKarpenterCreateServiceAccount()
```

Comment:
this feels... a bit odd. Shouldn't this be done before the call to `Install`? Like in `ctl` we load the config, validate it and then call `Create`. This feels like it falls under the validation/defaulting part

Reply:
Yeah I could call it there I guess. In the validator maybe?

Reply:
yeah. We have a `defaulter` logic somewhere, would fit in there. might be a sub part of validater